src/core/datastore/utils/utils.js

```diff
@@ -1,9 +1,16 @@
 import _isEmpty from 'lodash/isEmpty';
 
+import { KinveyError } from '../../errors';
 import { isNonemptyString } from '../../utils';
 
 export const dataStoreTagSeparator = '.';
 
+export function getEntitiesPendingPushError(entityCount, prefix) {
+  const countMsg = `There are ${entityCount} entities, matching this query or id, pending push to the backend.`;
```

Comment:
Lets use an `if` statement to make the count message more grammatically correct.

```javascript
let countMsg = `There are ${entityCount} entities, matching the provided query, pending push to the backend.`;

if (entityCount === 1) {
  countMsg = `There is ${entitiyCount} entity, matching the provided query or id, pending push to the backend.`;
}
```